internal/provider/provider.go

```diff
@@ -234,25 +251,30 @@ func (p *IncusProvider) Configure(ctx context.Context, req provider.ConfigureReq
 	// in Terraform configurations where the Incus remote might not
 	// exist yet.
 	for _, remote := range data.Remotes {
-		port := remote.Port.ValueString()
-		if port == "" {
-			port = "8443"
+		isDefault := false
+
+		protocol := remote.Protocol.ValueString()
+		if protocol == "" {
+			protocol = "incus"
 		}
 
-		scheme := remote.Scheme.ValueString()
-		if scheme == "" {
-			scheme = "unix"
+		auth_type := remote.Auth_Type.ValueString()
```

Comment:
Please follow our naming convention (style and use complete words): `authenticationType` and `remote.AuthenticationType`